src/component/app.tsx

```diff
@@ -80,9 +81,10 @@ class App extends React.Component<AppProps> {
 		// Todo: project and page don't update on page change
 		const project = this.props.store.getCurrentProject();
 		const title = `${project && project.getName()}`;
+		const styleguide = this.props.store.getStyleguide() as Styleguide;
```

Comment:
do we need the type cast?  `getStyleguide` is typed to return `Styleguide`.

Reply:
That's no longer true, I changed that. The styleguide may be undefined now, opening a styleguide creates it. On the other hand, the patterns path is never undefined. I think it is more consistent if the styleguide is not set as long as no styleguide is open, and not that a styleguide with an invalid state exists.
However, in this case, styleguide can actually be undefined, so I should check that better.

Reply:
Done.